sublime-package.json

```diff
@@ -75,6 +75,11 @@
               },
               "markdownDescription": "The command to start the language server."
             },
+            "ClientUseNodeIpc": {
+              "type": "boolean",
+              "default": false,
+              "markdownDescription": "Communicate with the language server over Node.js IPC. This lets the server print to stdout without disrupting the LSP communication. It's non-standard, but is used by VSCode. The command must be adjusted accordingly, e.g. `--stdio` must be replaced with `--node-ipc` in case of vscode-eslint. `tcp_port` is ignored if this is enabled."
```

Comment:
Too verbose?

Reply:
Yes :)
maybe reduce it to not mention some specificty like VS Code, or LSP-eslint
```
"Communicate with the language server over Node.js IPC. The command might be adjusted accordingly, e.g. `--stdio` must be replaced with `--node-ipc` for some servers."
```